src/modules/Thread/context/hooks/useSendFileMessage.ts

```diff
@@ -50,37 +50,42 @@ export default function useSendFileMessageCallback({
       const params = onBeforeSendFileMessage?.(file, quoteMessage) ?? createParamsDefault();
       logger.info('Thread | useSendFileMessageCallback: Sending file message start.', params);
 
-      currentChannel?.sendFileMessage(params)
-        .onPending((pendingMessage) => {
+      if (currentChannel == null) {
+        logger.warning('Thread | useSendFileMessageCallback: currentChannel is null. Skipping file message send.');
+        resolve(null);
```

Comment:
Same for this file change. Just wrapped with if - else for the better unit testing.

Reply:
Same with the above